android/app/src/main/java/app/organicmaps/widget/placepage/PlacePageView.java

```diff
@@ -646,7 +647,8 @@ private void refreshDetails()
     final String cap = mMapObject.getMetadata(Metadata.MetadataType.FMD_CAPACITY);
     refreshMetadataOrHide(!TextUtils.isEmpty(cap) ? getString(R.string.capacity, cap) : "", mCapacity, mTvCapacity);
 
-    refreshMetadataOrHide(mMapObject.hasAtm() ? getString(R.string.type_amenity_atm) : "", mAtm, mTvAtm);
+    refreshMetadataOrHide(mMapObject.hasAtm() ? getString(app.organicmaps.sdk.R.string.type_amenity_atm) : "", mAtm,
+                          mTvAtm);
```

Comment:
Better ```import app.organicmaps.sdk.R``` ?
Here and above.

Reply:
Not better
This file already imports app.om.R
We can't import different class with same name
That's the only way